speech/check_dataVals.m

```diff
@@ -396,13 +462,20 @@ function update_plots(src,evt)
         set(UserData.warnPanel,'HighlightColor','yellow')
         set(UserData.warnText,'String',outstring)
         pause(0.0001)
-        [UserData.htracks,UserData.hsub] = plot_rawFmtTracks(UserData.dataVals,grouping,UserData.trialset,UserData.plotPanel,UserData.expt);
+        [UserData.htracks,UserData.hsub] = plot_rawAcoustTracks(UserData.dataVals,grouping,UserData.trialset,UserData.plotPanel,UserData.expt,UserData.sigs2plot);
+        if strcmp(UserData.sigs2plot{1},'f0') %plot line at 40 ms for f0 data
```

Comment:
Do we want this if *any* of the sigs2plot is f0?

Reply:
Sure. Addressed in 6ecc0797d0844b9f21d921d92863748732d28638